pkgs/tools/video/xjadeo/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenv, autoreconfHook, fetchFromGitHub, ffmpeg, freetype, libGLU
+, libjack2, liblo, libX11, libXv, pkg-config, portmidi, xorg }:
+stdenv.mkDerivation {
+  pname = "xjadeo";
+  version = "0.8.10";
+
+  src = fetchFromGitHub {
+    owner = "x42";
+    repo = "xjadeo";
+    rev = "c56548cad7b9bd358eb2c776de5ec5323f41006e";
+    sha256 = "0cwh3zimrysyank3x1bcysdz59f8srqdca7jwv97ri3y63dlhifj";
+  };
+
+  nativeBuildInputs = [
```

Comment:
Are those packages all required to build the package or are they runtime deps?

Reply:
They are listed under "build dependencies" in the README, but as mentioned in the comment the last few are recommended optional deps. Seeing as they're recommended I've included them for now.

On this topic, I'm not sure if these would be better split between `nativeBuildInputs` and `buildInputs`? The distinction between the two is still a bit fuzzy in my mind - I'll check the manual again.

Reply:
Everything that is executed the host goes into nativeBuildInputs.